README.md

```diff
@@ -29,6 +29,11 @@ Call `katex.render` with a TeX expression and a DOM element to render into:
 katex.render("c = \\pm\\sqrt{a^2 + b^2}", element);
 ```
 
+Note: If you don't want to double backslack to escape, you can use `String.raw`
```

Comment:
Shouldn't that be "double backslash"? Or is this a word I don't know?

Reply:
If it is, I don't know it either. Apparently I'm trying to add subliminal messaging for slack.

Reply:
Industrial sponsorship! 😄